### PR TITLE
Fix broken links to pixhawk.org

### PR DESCRIPTION
**Pull request by hamishwillee**
This fixes broken links to pixhawk.org (either never existed or removed in latest update). 

There remain some open issues:
1. In \Firmware\README.md we have link to https://pixhawk.org/modules/stm32f4discovery - is the discovery kit still available, used, or useful? If not will delete that line.
2. There are links in \Tools\px_uploader.py and \Firmware\src\systemcmds\ver\ver.c to silicon errata doc (which says).
```
FMUv2 / Pixhawk Silicon Errata
STM32F427VIT6 (errata)
Flash Bank 2 and full speed USB device exclusive
Silicon revisions up to rev 1 (revision 3 is the first not affected) can produce errors / data corruption when accessing the 2nd flash bank while there is activity on PA12, which is one of the USB data lines. There is no workaround / software fix for this, except to not use the flash bank #2. Since USB is needed to program the device, Pixhawk revisions built with silicon revisions < rev 3 can only use up to 1MB of the 2MB flash of the microprocessor.
```
Should that section just go in the Pixhawk 1 doc, or do we need a whole specific topic for silicon errata? If we need a unique topic can that go in the user guide, or would it need to go in pixhawk.org?

3.  `\src\modules\systemlib\circuit_breaker.h` Line 47: 
```
* SAFETY WARNING  --  SAFETY WARNING  --  SAFETY WARNING
 *
 * OBEY THE DOCUMENTATION FOR ALL CIRCUIT BREAKERS HERE,
 * ENSURE TO READ CAREFULLY ALL SAFETY WARNINGS.
 * http://pixhawk.org/dev/circuit_breakers
 *
 * CIRCUIT BREAKERS ARE NOT PART OF THE STANDARD OPERATION PROCEDURE
 * AND MAY DISABLE CHECKS THAT ARE VITAL FOR SAFE FLIGHT.
 */
```
But the line "http://pixhawk.org/dev/circuit_breakers" just points generically to the devguide.
Perhaps point it to the devguide parameter reference?

### CODE REVIEW

**Comment by dagar** [comment]
I'm not sure where appveyor is coming from.

**Comment by hamishwillee** [comment]
Me either. This is pulling in some unintended changes in the parameters documentation - though they do look useful :-)

**Comment by hamishwillee** [comment]
@bkueng Do you have an opinion on the silicon errata question (2) in the above description?

**Comment by bkueng** [comment]
Re 2: I think it's enough to have it in the Pixhawk 1 docs.

**Comment by LorenzMeier** [comment]
Needs a rebase.

File: README.md [hunk 1]
```diff
@@ -92,12 +92,12 @@ This repository contains code supporting these boards:
...
     * [Pixhawk Mini](https://docs.px4.io/en/flight_controller/pixhawk_mini.html)
     * [Pixfalcon](https://docs.px4.io/en/flight_controller/pixfalcon.html)
-  * FMUv3.x [Pixhawk 2](https://pixhawk.org/modules/pixhawk2)
+  * FMUv3.x [Pixhawk 2](https://docs.px4.io/en/flight_controller/pixhawk-2.html)
   * FMUv4.x
     * [Pixracer](https://docs.px4.io/en/flight_controller/pixracer.html)
     * [Pixhawk 3 Pro](https://docs.px4.io/en/flight_controller/pixhawk3_pro.html)
   * FMUv5.x (ARM Cortex M7, future Pixhawk)
-  * [STM32F4Discovery](http://www.st.com/en/evaluation-tools/stm32f4discovery.html) (basic support) [Tutorial](https://pixhawk.org/modules/stm32f4discovery)
+  * [STM32F4Discovery](http://www.st.com/en/evaluation-tools/stm32f4discovery.html) (basic support) [Tutorial](https://pixhawk.org/modules/stm32f4discovery)
```
Comment (LorenzMeier) [inline README.md]: Whitespace

File: src/modules/commander/commander_params.c [hunk 2]
```diff
@@ -300,11 +300,12 @@ PARAM_DEFINE_INT32(COM_ARM_SWISBTN, 0);
...
  * @value 0 Warning
- * @value 1 Return to land
- * @value 2 Land at current position
- * @value 3 Return to land at critically low level, land at current position if reaching dangerously low levels
+ * @value 1 Return mode
+ * @value 2 Land mode
+ * @value 3 Return mode at critically low level, Land mode at current position if reaching dangerously low levels
  * @decimal 0
  * @increment 1
+ * @increment 1
```
Comment (lamping7) [inline src/modules/commander/commander_params.c]: duplicate `@increment`

**Comment by hamishwillee** [comment]
@bkueng FYI, I ended up adding Silicon Errata in its own doc - this is more "extensible".

Any thoughts on 
1. In \Firmware\README.md we have link to https://pixhawk.org/modules/stm32f4discovery - is the discovery kit still available, used, or useful? If not will delete that line.
2.  In text below there is a link to http://pixhawk.org/dev/circuit_breakers  that just points to the devguide. Do we need a circuitbreakers topic?

```
\src\modules\systemlib\circuit_breaker.h Line 47:
* SAFETY WARNING  --  SAFETY WARNING  --  SAFETY WARNING
 *
 * OBEY THE DOCUMENTATION FOR ALL CIRCUIT BREAKERS HERE,
 * ENSURE TO READ CAREFULLY ALL SAFETY WARNINGS.
 * http://pixhawk.org/dev/circuit_breakers
 *
 * CIRCUIT BREAKERS ARE NOT PART OF THE STANDARD OPERATION PROCEDURE
 * AND MAY DISABLE CHECKS THAT ARE VITAL FOR SAFE FLIGHT.
 */
```

**Comment by hamishwillee** [comment]
Closing. This is too messy to merge. I have rebuild the changes I want in https://github.com/PX4/Firmware/pull/9822

**Pull request closed without merging**